dance/transforms/cell_feature.py

```diff
@@ -30,8 +31,8 @@ class WeightedFeaturePCA(BaseTransform):
 
     _DISPLAY_ATTRS = ("n_components", "split_name", "feat_norm_mode", "feat_norm_axis")
 
-    def __init__(self, n_components: int = 400, split_name: Optional[str] = None, feat_norm_mode: Optional[str] = None,
-                 feat_norm_axis: int = 0, **kwargs):
+    def __init__(self, n_components: Union[float, int] = 0.9, split_name: Optional[str] = None,
```

Comment:
About MRE in imputation :https://pubmed.ncbi.nlm.nih.gov/33991694/

Reply:
1 about MRE:Hao M, Gong J, Zeng X, et al. Large scale foundation model on single-cell transcriptomics[J]. bioRxiv, 2023: 2023.05. 29.542705.

Reply:
Secondly,because all models need to be saved, otherwise the algorithm will report an error. When the model output is always nan, the model cannot be saved.

Reply:
Third, I am currently determining the default parameters of the function just like the message I sent in WeChat.